maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -310,6 +343,57 @@ private void downloadOWAs(File targetDirectory, DistroProperties distroPropertie
 		}
 	}
 
+	/**
+	 * Sets the configuration folder for the specified server using the provided distro properties.
+	 *
+	 * @param server           The server for which to set the configuration folder.
+	 * @param distroProperties The distro properties containing the configuration information.
+	 */
+	private void setConfigFolder(Server server, DistroProperties distroProperties) throws MojoExecutionException {
+		if(distroProperties.getConfigArtifacts().isEmpty()) {
+			return;
+		}
+		File configDir = new File(server.getServerDirectory(), SDKConstants.OPENMRS_SERVER_CONFIGURATION);
+		configDir.mkdir();
+		downloadConfigs(distroProperties, configDir);
+		File referenceApplicationFile = new File(configDir, "referenceapplication-distro.owa");
```

Comment:
Do you mind giving me some more details about the `referenceapplication-distro.owa` file? Is it the one that bundles the configuration files?

Reply:
Yes.